includes/query/SMW_QueryProcessor.php

```diff
@@ -442,7 +442,7 @@ static public function getResultPrinter( $format, $context = self::SPECIAL_PAGE
 	 *
 	 * @return IParamDefinition[]
 	 */
-	public static function getParameters( $context = null ) {
+	public static function getParameters( $resultPrinter = null, $context = null ) {
```

Comment:
This changes makes code that calls the function with $context wrong in a somewhat subtle manner. For this reason normally new parameters are added to the end of the list. Any reason why to not do that here?

Reply:
I checked the calls from SMW and there is only one that actually uses `$context` (which I fixed). The reason I put `$resultPrinter` in front was simply that the method is called more often with that than with `$context`. I did not think of external callers, though. I'll turn the params around.